cas-server-core/src/main/java/org/jasig/cas/CasVersion.java

```diff
@@ -43,4 +47,27 @@ private CasVersion() {
     public static String getVersion() {
         return CasVersion.class.getPackage().getImplementationVersion();
     }
+
+    /**
+     * Gets last modified date/time for the module.
+     * @return the date/time
+     */
+    public static DateTime getDateTime() {
```

Comment:
You're right, the default login page is mainly for us. I would be great to have it as well at startup.

+1 that way